DataFormats/Phase2TrackerDigi/interface/Phase2TrackerDigi.h

```diff
@@ -35,7 +35,7 @@ class Phase2TrackerDigi {
   unsigned int strip()   const { return row(); }
   unsigned int edge()    const { return column(); } // CD: any better name for that? 
   // Access to the (raw) channel number
-  unsigned int channel() const { return theChannel; }
+  unsigned int channel() const { return 0x7FFF & theChannel; }
```

Comment:
packedPosition() is fine, but could be dropped. It is now equivalent to channel() and not used outside this header file (as far as I can tell... i.e. using cmsglimpse). I just kept it in case private code are using it (not breaking the interface).